app/components/page_list_component/view.html.erb

```diff
@@ -40,28 +40,25 @@
                 </dt>
 
                 <dd class="govuk-summary-list__value">
-                  <p class="govuk-!-margin-2">
-                    <%= t("page_conditions.#{condition.validation_errors.length > 1 || condition.has_errors_for_field?(:answer_value) ? "condition_check_page_text_with_errors" : "condition_check_page_text"}", check_page_text: question_text_for_page(condition.check_page_id)) %>
+                  <ul class="govuk-list govuk-!-margin-0">
+                    <% condition.validation_errors.each do |error| %>
```

Comment:
Do we want use `<ul>` if there is only 1 validation error?

Reply:
I think this is OK and it's not worth complicating it - a list with one item is semantically valid, and this is what the Design System error summary component does which is probably the most similar existing component